config/index.js

```diff
@@ -312,6 +312,23 @@ var conf = convict({
     env: 'LOCKOUT_ENABLED',
     default: false
   },
+  oauth: {
+    host: {
+      doc: 'OAuth host for verifying tokens',
+      default: 'localhost',
+      env: 'OAUTH_HOST'
+    },
+    port: {
+      doc: 'OAuth port for veryfing tokens',
+      default: 9010,
+      env: 'OAUTH_PORT'
+    },
+    insecure: {
+      doc: 'Connect to OAuth server via insecure plain http',
+      default: false,
+      env: 'OAUTH_INSECURE'
+    }
+  },
```

Comment:
This assumes the oauth server is a process on the same machine?

I know in our dev servers, we seem to like to put several services behind paths as well.